src/components/shared/Search/search.module.scss

```diff
@@ -24,10 +23,6 @@
   opacity: 0.5;
 }
 
-.input:focus {
-  width: 250px;
-}
-
```

Comment:
why?

Reply:
Just removed it because transition effect breaks click on the search button
Didn't have time to discover the problem and solve it